pkg/interface/src/views/apps/chat/components/lib/group-item.js

```diff
@@ -1,13 +1,13 @@
 import React, { Component } from 'react';
 import { Link } from 'react-router-dom';
 import { ChannelItem } from './channel-item';
-import { deSig } from "~/logic/lib/util";
+import { deSig, cite } from "~/logic/lib/util";
 
 export class GroupItem extends Component {
   render() {
     const { props } = this;
     const association = props.association ? props.association : {};
-    const DEFAULT_TITLE_REGEX = new RegExp(`(( <-> )?~${deSig(window.ship)}( <-> )?)`);
+    const DEFAULT_TITLE_REGEX = new RegExp(`(( <-> )?~(?:${deSig(window.ship)}|${deSig(cite(window.ship))})( <-> )?)`);
```

Comment:
`cite` deSigs already iirc

Reply:
it returns a sig'd ship no matter what

Reply:
... well now I'm confused

Reply:
I don't know but I tested it on my planet and on a comet that had previously not exhibited correct behavior and now they both do

Reply:
so: 
- `window.ship` doesn't have a sig so `deSig` isnt really necessary
- `cite` will take a sig'd or desig'd ship and return a sig'd ship
- the DM format *does* use sigs on both ships

so maybe just remove `deSig` from window.ship and it will make more sense? since we want to force the sig in both contexts and we have to desig to have the start of the regex apply to the query regardless